src/akka/client.rs

```diff
@@ -12,6 +12,22 @@ pub fn get_actor_count(url: &String, timeout: u64) -> Result<u64, String> {
     get_actor_count_async(url, timeout)
 }
 
+pub fn get_deadletters(url: &String, window: u64) -> Result<(DeadLettersSnapshot, DeadLettersWindow), String> {
+    get_deadletters_async(url, window)
+}
+
+#[tokio::main]
+async fn get_deadletters_async(url: &String, window: u64) -> Result<(DeadLettersSnapshot, DeadLettersWindow), String> {
+    let url = format!("{}?window={}", url, window);
+    let response = reqwest::get(&url).await.map_err(|e| e.to_string())?;
+    if !response.status().is_success() {
```

Comment:
what about using if/else with a base case `is_success == true` and no `return` ?
```rust
if response.status().is_success() {
  Ok()
} else {
  Err()
}
```

Reply:
good suggestion, fixed here and in 2 other places :+1: